src/bentoml/server.py

```diff
@@ -189,6 +197,9 @@ def get_client(self) -> Client | None:
         pass
 
     def stop(self) -> None:
+        # NOTE: User shouldn't manually set this since this envvar will be managed by BentoML.
+        os.environ[BENTOML_SERVE_FROM_SERVER_API] = str(False)
```

Comment:
delete the env var instead of setting to false?

Reply:
👍